### ISSUE TITLE
The "fixed" class no longer positions correctly in Chrome

### ISSUE THREAD

**Issue by rajsite** [GoogleChrome/dialog-polyfill#220]
Using the provided `fixed` class in Chrome no longer centers the dialog in the middle of the page in Chrome. I think it is due to a [change in style](https://groups.google.com/a/chromium.org/g/blink-dev/c/Ow63WaHuTgc/m/T2Ehuv3aAAAJ) for dialog elements in Chrome.

Here is an [example page](https://dialog-polyfll-stylechange.glitch.me/) and a screenshot using Chrome 90 and Firefox 88.

Chrome 90:

![image](https://user-images.githubusercontent.com/1588923/116830614-1c563d00-ab71-11eb-9e37-7c2d68b3d0a0.png)

Firefox 88:

![image](https://user-images.githubusercontent.com/1588923/116830631-40b21980-ab71-11eb-8207-c816fdd0dedc.png)

It looks like the `dialog:-internal-modal` pseudoelement has `bottom: 0` set which is impacting the style:
![image](https://user-images.githubusercontent.com/1588923/116830868-e796b580-ab71-11eb-9779-004d6652a639.png)

**Comment by inopinatus** [comment]
agreed, native chrome gets it right, but looks like loading the 0.5.6 polyfill CSS intrudes on the native styles, and position goes sideways.

or rather, in my apps, up.

**Comment by IanVS** [comment]
Adding the following to my css seems to result in a decent positioning in chrome, firefox, and safari:

```
dialog.fixed {
  bottom: 50%;
  transform: none;
}
```

edit: nevermind, that breaks firefox, placing the modal too low.  Not sure how to work around this one...

**Comment by xtools-at** [comment]
try this
```
/* Chromium Only */
@supports (contain: paint) and (not (-moz-appearance: none)) {
    dialog.fixed {
        top: 0;
        transform: none;
    }
}
```
